src/proof/alf/alf_node_converter.cpp

```diff
@@ -721,7 +720,7 @@ bool AlfNodeConverter::isHandledSkolemId(SkolemFunId id)
     case SkolemFunId::RE_UNFOLD_POS_COMPONENT:
     case SkolemFunId::BAGS_DEQ_DIFF:
     case SkolemFunId::BAGS_DISTINCT_ELEMENTS:
-    case SkolemFunId::BAGS_MAP_PREIMAGE_INJECTIVE:
+    case SkolemFunId::BAGS_DISTINCT_ELEMENTS_INJECTIVE:
```

Comment:
This enum is specific for map operator, and it refers to a singleton preimage. The name distinct elements is not accurate here.